### ISSUE TITLE
Request: Why not expose getHome()

### ISSUE THREAD

**Issue by blobfish** [sago007/PlatformFolders#26]
Wondering why the class can't expose the getHome() function also?

**Comment by sago007** [comment]
My original thought was to not expose platform dependent files. Although I did start doing it with the "sago::internal::" namespace for the Windows wide char conversion functions, so I have started to relax the requirement.

I wonder if people really want to use the internal getHome function. It contains a workaround that is no longer needed since Ubuntu 18.04 reached EOL. That might confuse people.

**Comment by blobfish** [comment]
Sorry, I didn't phrase my question very good. Why not have a public function to get the users home directory?
[QDir](https://doc.qt.io/qt-5/qdir.html#homePath)
[QStandardPaths::HomeLocation](https://doc.qt.io/qt-5/qstandardpaths.html)

**Comment by totalgee** [comment]
Yes, this would be nice to just get the equivalent to the home (`~`) directory...it's the base of all the other directories you already expose with this library. I imagine it's "safe" to just get it by going one directory back (`..`) from, say, `getDownloadFolder()`, but this may not (?) always be the case. For example, if I were to use `getConfigHome()` on Windows 11, I would need to go back twice (`../..`), since it returns `~/AppData/Roaming/` on my machine.

**Comment by totalgee** [comment]
> Yes, this would be nice to just get the equivalent to the home (`~`) directory...it's the base of all the other directories you already expose with this library. I imagine it's "safe" to just get it by going one directory back (`..`) from, say, `getDownloadFolder()`, but this may not (?) always be the case. For example, if I were to use `getConfigHome()` on Windows 11, I would need to go back twice (`../..`), since it returns `~/AppData/Roaming/` on my machine.

I guess this would return the lookup of `FOLDERID_Profile` for Windows, and your existing internal `getHome()` function for Mac/Linux...
